### PR TITLE
Skeleton Code for adding Azure Monitor exporter to OC Agent

### PR DESCRIPTION
**Pull request by ChrisCoe**
I unmarshel the viper following the format of all the other exporters in this package. The mod and sum files are automatically generated with my imports.

@reyang @lzchen

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,6 +8,7 @@ require (
 	contrib.go.opencensus.io/exporter/stackdriver v0.12.3-0.20190626200219-09504ed717c7 // TODO: pin a released version
 	contrib.go.opencensus.io/exporter/zipkin v0.1.1
 	contrib.go.opencensus.io/resource v0.1.1
+	github.com/ChrisCoe/opencensus-go-exporter-azuremonitor v0.0.0-20190711172825-09f39f7ce008
```
Comment (reyang) [inline go.mod:11]: minor, might be good to use a non-zero version number

File: config.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+receivers:
```
Comment (reyang) [inline config.yaml]: Is this just for demo purpose?

**Review by reyang — APPROVED** [review]
LGTM, not sure if we want to include the `config.yaml` though.

File: config.yaml (hunk 2 above)
Comment (ChrisCoe) [reply config.yaml]: Yes. I will take out the config file as each developer would have to create their own.

**Pull request merged**